pom.xml

```diff
@@ -90,7 +90,9 @@
                 </executions>
                 <configuration>
                     <excludes>
-                        <exclude>**/Main*</exclude>
+                        <exclude>**/src/main/java/core/basesyntax/Main.java*</exclude>
+                        <exclude>**/FruitTest.java*</exclude>
```

Comment:
what is FruitTest and why we exclude it? What about Fruit model, maybe worth to exclude it too?